### ISSUE TITLE
Add color to visualized credible interval

### ISSUE THREAD

**Issue by aaronyan** [Automattic/abacus#778]
Separating this subtask from https://github.com/Automattic/abacus/issues/660 into its own issue. Color would be used to help with the metric interpretation instead of using color to signal deployment action. Here's a mockup!

- > min diff -> green
- < min diff -> blue
- between min diff -> yellow

<img width="1680" alt="186020501-6d55891b-29be-4979-b5c1-a88d77a237a6" src="https://user-images.githubusercontent.com/4505888/189364683-c32ea030-ae05-49b7-8fc2-0d165eb95b5a.png">

**Comment by aaronyan** [comment]
Hmm this might be a bit too much color. 😅 I'm proposing now to make the whole bar a certain color depending on the **recommendation**.

<img width="1680" alt="Screen Shot 2022-09-15 at 2 56 16 PM" src="https://user-images.githubusercontent.com/4505888/190515856-5d76f37f-24d3-4cb1-bf59-86beecb45943.png">

- The color is based on the recommendation, not analysis
- When the recommendation is "not enough certainty", then we color the bar grey
- Example: Analysis says "control ahead" but the recommendation is "not enough certainty", then we color the bar grey

**Comment by aaronyan** [comment]
The multi-colored bars might be a bit distracting/ confusing. Here's what it would look like mocked-up versus the single color bar!

<img width="1392" alt="Screen Shot 2022-09-15 at 4 14 22 PM" src="https://user-images.githubusercontent.com/4505888/190524573-108f6a14-67a9-4c7b-91e9-80a51c770a5f.png">

<img width="1392" alt="Screen Shot 2022-09-15 at 4 14 29 PM" src="https://user-images.githubusercontent.com/4505888/190524580-d46eaade-2273-4589-9d23-88dc2aa178c7.png">

**Comment by jessie-ross** [comment]
I'm not particularly a fan of this use of colour, it's too easy to ignore the greyed out graphs, and since the decision on whether to colour them or not isn't smart enough an experimenter may accidentally ignore important information that we have greyed out.

I really like the new column arrangment, It is a great way to fit abs/relative/lift all at once :)

**Comment by razvanpapadopol** [comment]
> an experimenter may accidentally ignore important information that we have greyed out

Yup, once we introduce color, grey becomes tricky. An option would be to use only the 2 colors from the original proposal. 

<img width="160" alt="Screen Shot 2022-09-16 at 17 43 35" src="https://user-images.githubusercontent.com/14192054/190666129-7ee82e9d-d93c-4776-90c6-499fbd551fad.png">

**Comment by aaronyan** [comment]
Thanks for the feedback y'all! I'm down to use the 2 color option from the original proposal:

- Green (`#00d084`) – Enough data is available to make a deployment recommendation.
- Yellow (`#fcb900`) – More data is required before we can make a deployment recommendation.

<img width="1125" alt="156829388-85625ccc-05c0-45df-984a-aee4f62c5620" src="https://user-images.githubusercontent.com/4505888/190681076-fe13f5c5-116d-42ee-9f50-8a5bcb40bffb.png">

<img width="1900" alt="Screen Shot 2022-09-16 at 9 02 03 AM" src="https://user-images.githubusercontent.com/4505888/190681562-55e41975-e58b-41d3-9792-ef344b054385.png">
